oop_Session1/CompositeElement.java

```diff
@@ -0,0 +1,54 @@
+package oop_Session1;
+
+import java.util.List;
+import java.util.ArrayList;
+/**
+ * This class is responsible for initializing Composite element i.e. the elements which have more tags inside it. 
+ * @author Akshita Dixit
+ */
+public class CompositeElement extends Element{
+	
+	// List stores tags inside parent tag
+	List<Element> element = new ArrayList<Element>();
+
+	/**
+	 * constructor to initialization of values
+	 * @param id
+	 * @param className
+	 */
+	public CompositeElement(String id, String className) {
+		
+		// calling parent class for initialization
+		super(id,className);
+	}
+	
+	 /**
+	  *  method for adding Tag
+	  * @param tag - object of element
+	  */
+	void add(Element tag){
+		
+		// Adding tag or element
+		element.add(tag);
+	}
+	
+	
+	/**
+	 * Returns list of tags inside parent tag
+	 * @return list of elements
+	 */
+	List<Element> tagList(){

```

Comment:
The name of the return type should be consistent with the method name, for "getter"-type methods. For example: in this case, either the method should be named: `getElements` or the class should be called `Tag` instead of `Element`. Given that the problem mentions the class should be `Element`, it is the method name that should be changed.